### PR TITLE
Added an email script to handle form from contact-page. Add a feature…

### PR DESCRIPTION
**Pull request by JesperZamora**
… that will show the user that the email is about to be send and afterwards confirm wether the message was sent or not + added som CSS to that feature

### CODE REVIEW

File: js/email_script.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+const form = document.querySelector('form');
+const display = document.querySelector('.display_message');
+const urlMime = 'http://localhost:8080/send-mail';
```
Comment (JesperNielsen99) [inline js/email_script.js:3]: Skal det være localhost:8080?

File: js/email_script.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+form.addEventListener('submit', (e) => {
+    e.preventDefault();
+
+    const data = new FormData(e.currentTarget);
+    console.log(data.get(''))
```
Comment (JesperNielsen99) [inline js/email_script.js:9]: skal ikke logges

File: js/email_script.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    const entries= Object.fromEntries(data.entries());
+    console.table(entries);
```
Comment (JesperNielsen99) [inline js/email_script.js:11]: Skal ikke logges

**Review by JesperNielsen99 — CHANGES_REQUESTED** [review]
Minor changes to be made.

**Comment by JesperZamora** [comment]
Skulle være ændret nu

**Review by KEATLLoevkilde — APPROVED** [review]

**Review by JesperNielsen99 — APPROVED** [review]

**Pull request merged**